feature-swap-impl/src/main/java/io/novafoundation/nova/feature_swap_impl/domain/swap/RealSwapService.kt

```diff
@@ -76,6 +76,12 @@ internal class RealSwapService(
         isCustomFeeToken && exchange.canPayFeeInNonUtilityToken(asset) && currentMetaAccount.type.requestedAccountPaysFees()
     }
 
+    override suspend fun sync(coroutineScope: CoroutineScope) {
+        exchanges(coroutineScope)
+            .values
+            .forEach { it.sync() }
```

Comment:
Lets make it async so some slow source wont stop everything else from syncing 
We have `mapAsync` helper for that